cmd/samproxy/main.go

```diff
@@ -35,6 +36,7 @@ type Options struct {
 	RulesFile  string `short:"r" long:"rules_config" description:"Path to rules config file" default:"/etc/samproxy/rules.toml"`
 	PeerType   string `short:"p" long:"peer_type" description:"Peer type - should be redis or file" default:"file"`
 	Version    bool   `short:"v" long:"version" description:"Print version number and exit"`
+	Debug      bool   `short:"d" long:"debug" description:"If enabled, runs debug service on port 6060"`
```

Comment:
specifying port 606 is not quite true right as it will try to find a port higher than 6060 if it's in use?